test/plugins/package.json

```diff
@@ -10,8 +10,8 @@
   "type": "module",
   "dependencies": {
     "@open-wc/semantic-dom-diff": "^0.20.1",
-    "@vaadin/testing-helpers": "^1.0.0",
```

Comment:
We don't need this dev dependency in the `plugins` package, it was added by mistake.